File: C# ad y ad/2_ Orientado a objetos/S05.2_ Consolas/Solucion_consolas/Consolas/Src/Consola/Consola.cs

namespace Consolas
{
	class Consola
	{
		int volumen;
		int brillo;


		public Consola()
		{
			volumen = 100;
			brillo = 100;
		}


		public int GetVolumen()
		{
			return volumen;
		}
		public int GetBrillo()
		{
			return brillo;
		}

		public void SetVolumen(int volumen)
		{
			this.volumen = volumen;
		}
		public void SetBrillo(int brillo)
		{
			this.brillo = brillo;
		}
	}
}

File: C++ ad y ad/3_ Orientado a objetos/S05.2_ Consolas/Solucion_consolas/Consolas/src/consola/consola.cpp
#include "consola.h"

#include <iostream>

using namespace std;


Consola::Consola()
{
	volumen = 100;
	brillo = 100;
}
Consola::~Consola()
{
	cout << "La consola ha sido liberada de la memoria." << endl << endl;
}


int Consola::getVolumen()
{
	return volumen;
}
int Consola::getBrillo()
{
	return brillo;
}

void Consola::setVolumen(int volumen)
{
	this->volumen = volumen;
}
void Consola::setBrillo(int brillo)
{
	this->brillo = brillo;
}
